test/e2e/util.go

```diff
@@ -178,7 +193,7 @@ func waitForPodsRunningReady(ns string, minPods int, timeout time.Duration) erro
 	}
 	Logf("Waiting up to %v for all pods (need at least %d) in namespace '%s' to be running and ready",
 		timeout, minPods, ns)
-	for start := time.Now(); time.Since(start) < timeout; time.Sleep(podPoll) {
+	for start := time.Now(); time.Since(start) < timeout; time.Sleep(poll) {
```

Comment:
Agh, the correct way to wait in e2es keeps changing under me! Will change this :-)